JarrydWorland/2021_Capstone_GP2_Team
# Level prefabs

This branch adds:
Rooms of every single combination of directions
New rooms with oil spills and spikes
New rooms filled with obstacles

I also moved rooms that don't have sprites for the hazards into a different folder

To test:
Play the game and try to go to every room
or set the starting room to the room you want to test
Wall sprites should also be visible as you play

I am not sure if I should just list every room here to make sure it is tested, but here goes
- [x] Obstacles_1
- [x] Obstacles_2
- [x] Obstacles_3
- [x] Obstacles_4
- [x] Obstacles_5
- [x] Oil_1
- [x] Oil_2
- [x] Spikes_1
- [x] Spikes_2
- [x] E
- [x] EW
- [x] N
- [x] NES
- [x] NS
- [x] S
- [x] SWN
- [x] W
- [x] WN
- [x] Check to make sure walls have the sprite and are not just pure white
